src/guide/buy_btc.md

```diff
@@ -0,0 +1,85 @@
+# Buy Bitcoin
+
+Todo add description
+
+<custom-tabs category="lang">
+
+<div slot="title">Rust</div>
+<section>
+
+```rust
+// TODO add docs
+```
+
+</section>
+
+<div slot="title">Swift</div>
+<section>
+
+```swift
+do {
+  let openingFeeParam:OpeningFeeParams? = nil
+  let req = BuyBitcoinRequest(
+    provider: .moonpay, openingFeeParams: openingFeeParam)
```

Comment:
Do we have to pass the openingFeePrams? Isn't the swift use the default value?

Reply:
Yes it does. Updated